symfony/plugins/orangehrmAdminPlugin/test/Dao/LanguageDaoTest.php

```diff
@@ -16,126 +16,110 @@
+     */
+    protected function setUp(): void
+    {
+        $this->languageDao = new LanguageDao();
+        $this->fixture = Config::get('ohrm_plugins_dir') . '/orangehrmAdminPlugin/test/fixtures/LanguageDao.yml';
+        TestDataService::populate($this->fixture);
+    }
 
-    public function testAddLanguage() {
-        
+    public function testAddLanguage(): void
+    {
         $language = new Language();
         $language->setName('Tamil');
-        
         $this->languageDao->saveLanguage($language);
-        
-        $savedLanguage = TestDataService::fetchLastInsertedRecord('Language', 'id');
-        
+        $savedLanguage = TestDataService::fetchLastInsertedRecord('Language', 'a.id');
         $this->assertTrue($savedLanguage instanceof Language);
         $this->assertEquals('Tamil', $savedLanguage->getName());
-        
     }
-    
-    public function testEditLanguage() {
-        
+
+    public function testEditLanguage(): void
+    {
         $language = TestDataService::fetchObject('Language', 3);
         $language->setName('Canadian French');
-        
         $this->languageDao->saveLanguage($language);
-        
-        $savedLanguage = TestDataService::fetchLastInsertedRecord('Language', 'id');
-        
+        $savedLanguage = TestDataService::fetchLastInsertedRecord('Language', 'a.id');
         $this->assertTrue($savedLanguage instanceof Language);
         $this->assertEquals('Canadian French', $savedLanguage->getName());
-        
     }
-    
-    public function testGetLanguageById() {
-        
+
+    public function testGetLanguageById(): void
+    {
         $language = $this->languageDao->getLanguageById(1);
-        
         $this->assertTrue($language instanceof Language);
         $this->assertEquals('Spanish', $language->getName());
-        
     }
-    
-    public function testGetLanguageList() {
-        
-        $languageList = $this->languageDao->getLanguageList();
-        
-        foreach ($languageList as $language) {
-            $this->assertTrue($language instanceof Language);
-        }
-        
-        $this->assertEquals(3, count($languageList));        
-        
-        /* Checking record order */
-        $this->assertEquals('English', $languageList[0]->getName());
-        $this->assertEquals('Spanish', $languageList[2]->getName());
-        
+
+    public function testGetLanguageList(): void
+    {
+        $languageFilterParams = new LanguageSearchFilterParams();
+        $result = $this->languageDao->getLanguageList($languageFilterParams);
+        $this->assertCount(3, $result);
+        $this->assertTrue($result[0] instanceof Language);
     }
-    
-    public function testDeleteLanguages() {
-        
-        $result = $this->languageDao->deleteLanguages(array(1, 2));
-        
+
+    public function testDeleteLanguages(): void
+    {
+        $toTobedeletedIds = [1, 2];
+        $result = $this->languageDao->deleteLanguages($toTobedeletedIds);
```

Comment:
can you add test assertion with empty array

Reply:
```
$nullValue = [];
$result = $this->languageDao->deleteLanguages($nullValue);
$this->assertEquals(0, $result);
```
you mean something like this right ?